src/index.html

```diff
@@ -1,13 +1,97 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/logo.png"
+          type="image/x-icon">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="bose logo">
+        </a>
+        <div class="top-bar__icons">
+          <a class="icon--phone-num">+1 234 5555 55 55</a>
```

Comment:
Usage of anchor tag for the phone number without 'href' property. You should use 'tel:' inside 'href' for phone numbers, like this: '<a href='tel:+12345555555'>+1 234 5555 55 55</a>'.